Dockerfile

```diff
@@ -0,0 +1,53 @@
+FROM node:slim as builder
+
+WORKDIR /usr/src/app
+
+COPY package.json yarn.lock ./
+RUN yarn install --frozen-lockfile
+
+COPY . ./
+RUN npx lerna bootstrap && yarn build
+
+FROM node:slim as broker
+
+WORKDIR /usr/src/app
+
+COPY packages/broker/package.json packages/broker/yarn.lock ./
+RUN yarn install --frozen-lockfile --production
+
+COPY --from=builder /usr/src/app/packages/broker/lib ./lib
+
+CMD [ "yarn", "start" ]
+
+FROM node:slim as core
+
+WORKDIR /usr/src/app
+
+COPY packages/core/package.json packages/core/yarn.lock ./
+RUN yarn install --frozen-lockfile --production
+
+COPY --from=builder /usr/src/app/packages/core/lib ./lib
+
+CMD [ "yarn", "start" ]
+
+FROM node:slim as service
```

Comment:
The service library will be used as an attachment to the existing GraphQL services, so it should be planned ad package to the already established GraphQL ms. I will add Docs about it this week

Reply:
Everything else is great so far, thanks!